R/build_url.r

```diff
@@ -27,3 +27,16 @@ build_url <- function(company_domain = .get_company_name(),
   url <- glue::glue("{base_url}/{company_domain}/{api_version}")
   return(url)
 }
+

```

Comment:
we removed these 3 functions in one of the other PRs, so can drop these from this one as well